docs/platform/concepts/byoc.rst

```diff
@@ -1,18 +1,19 @@
-Bring your own cloud (BYOC)
-===========================
+Bring your own cloud (BYOC) |beta|
+==================================
 
-Aiven services are usually deployed on Aiven managed infrastructure, using
-Aiven managed security protocols, and backed by Aiven managed storage and backups.
-This provides the most seamless, straight forward, and de-risked approach to deploying
-Aiven services. However, there are cases where this approach is not appropriate, such
-as the need to achieve strict regulatory compliance.
+.. topic:: Availability of BYOC
 
-In cases like these, Aiven offers customers the ability to instead BYOC (Bring
-Your Own Cloud).  BYOC allows customers to manage their own infrastructure,
-their own security posture and keep their data in their own cloud.
+   This is a beta feature, and its availability is limited. To enable BYOC for your Aiven organization, contact `sales@Aiven.io <mailto:sales@Aiven.io>`_.
 
-When to consider bringing your own cloud
-----------------------------------------
+About custom clouds
+-------------------
+
+Aiven services are usually deployed on Aiven-managed infrastructure, using Aiven-managed security protocols, and backed by Aiven-managed storage and backups. This provides the most seamless straightforward de-risked approach to deploying Aiven services. However, this *one-size-fits-all* approach may be not optimal for all business needs and project requirements, for example, when seeking a strict regulatory compliance.
+
+This is where the bring your own cloud (BYOC) feature comes in. With BYOC, Aiven offers you the ability to use your own cloud infrastructure (custom clouds) instead of using the Aiven-managed infrastructure. The BYOC feature allows you to manage your own infrastructure and security posture while keeping your data in your own cloud.
```

Comment:
> instead of using the Aiven-managed infrastructure.

That's a bit not correct.
I believe here we should have something like that:
"Aiven offers you the ability to manage the resources in your own cloud infrastructure (custom clouds) as they would have been deployed into the cloud infrastructure owned by Aiven." 
But I'd rather consult with others because I may be incorrect in some wording too

Reply:
@akudiyar, the 'as they would' suggestion is confusing to me. What are you trying to say here, exactly, that is different to the original suggestion?

Reply:
To me it looks like we do not offer management service for the resources in the customer infrastructure. But probably I'm not getting it right. We can close that discussion.